CoreImportExportPlugins/release/modules/ext/docs/CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/import-from-database.md

```diff
@@ -36,11 +36,37 @@ Importer" />
 The following is a high level workflow an analyst can follow (explained
 in detail later on):
 
-1.  Add the relevant JDBC Driver via "Manage Connections" -> "Drivers"
-    tab.
-2.  Add the Connection details in "Manage Connections" -> "Connections"
-    tab. If the username and password are not required, leave them
-    blank.
+1.  Add the relevant JDBC Driver via "Manage Connections" -&gt; "Drivers"
+    tab. These drivers will be specific to the database that you are
+    connecting to. You only need one driver for each database type e.g.
+    MySQL, PostgresQL, SQLite Example drivers:
```

Comment:
wondering if either Postgres should be removed here, or example driver for it be added

Reply:
I've added postgres to the list of example drivers as there is already a connection string mentioned and it's large popularity it's likely to be used.